presto-docs/src/main/sphinx/language/types.rst

```diff
@@ -80,6 +80,9 @@ CHAR
 ----
 
     Fixed length character data. Char type without length specified has default length of 1.
+    Comparison semantics is that of VARCHAR type (leading and trailing spaces are relevant).
```

Comment:
CHAR(x) has always x characters. So CHAR(7) `dog` is actually `dog    ` with 4 implicit trailing spaces. I will update this doc